cadc-vosi/src/main/java/ca/nrc/cadc/vosi/CapGetAction.java

```diff
@@ -0,0 +1,202 @@
+*  any later version.                   toute version ultérieure.
+*
+*  OpenCADC is distributed in the       OpenCADC est distribué
+*  hope that it will be useful,         dans l’espoir qu’il vous
+*  but WITHOUT ANY WARRANTY;            sera utile, mais SANS AUCUNE
+*  without even the implied             GARANTIE : sans même la garantie
+*  warranty of MERCHANTABILITY          implicite de COMMERCIALISABILITÉ
+*  or FITNESS FOR A PARTICULAR          ni d’ADÉQUATION À UN OBJECTIF
+*  PURPOSE.  See the GNU Affero         PARTICULIER. Consultez la Licence
+*  General Public License for           Générale Publique GNU Affero
+*  more details.                        pour plus de détails.
+*
+*  You should have received             Vous devriez avoir reçu une
+*  a copy of the GNU Affero             copie de la Licence Générale
+*  General Public License along         Publique GNU Affero avec
+*  with OpenCADC.  If not, see          OpenCADC ; si ce n’est
+*  <http://www.gnu.org/licenses/>.      pas le cas, consultez :
+*                                       <http://www.gnu.org/licenses/>.
+*
+************************************************************************
+*/
+
+package ca.nrc.cadc.vosi;
+
+import ca.nrc.cadc.net.HttpTransfer;
+import ca.nrc.cadc.reg.AccessURL;
+import ca.nrc.cadc.reg.Capabilities;
+import ca.nrc.cadc.reg.CapabilitiesReader;
+import ca.nrc.cadc.reg.CapabilitiesWriter;
+import ca.nrc.cadc.reg.Capability;
+import ca.nrc.cadc.reg.Interface;
+import ca.nrc.cadc.reg.client.LocalAuthority;
+import ca.nrc.cadc.reg.client.RegistryClient;
+import ca.nrc.cadc.rest.InlineContentHandler;
+import ca.nrc.cadc.rest.RestAction;
+import ca.nrc.cadc.rest.SyncOutput;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URL;
+import java.util.Set;
+import java.util.TreeSet;
+
+import org.apache.log4j.Logger;
+
+/**
+ *
+ * @author pdowler
+ */
+public class CapGetAction extends RestAction {
+    private static final Logger log = Logger.getLogger(CapGetAction.class);
+
+    /**
+     * Enable transformation of the capabilities template (default: true). Subclasses
+     * may disable this according to some policy. The current transform is to change
+     * the host name in every accessURL in the capabilities to match the host name used
+     * in the request. This works fine in most cases but would not work
+     * if some accessURL(s) within an application are deployed on a different host.
+     * For example, if the VOSI-availability endpoint is deployed on an separate host
+     * so it can probe the service from the outside, then capabilities transform
+     * would need to be disabled.
+     */
+    protected boolean doTransform = true;
+    
+    /**
+     * Enable injection of additional capabilities for external authentication
+     * providers (default: true).
```

Comment:
It looks like the default is actually false

Reply:
yeah, that is code for the thing I was toying with for IVOA but wanted to leave it disabled for now. Will fix comment.